Code/Framework/AzCore/Tests/AZTestShared/Utils/Utils.cpp

```diff
@@ -91,6 +91,30 @@ namespace UnitTest
     {
         return SuppressExpectedErrors(window, message);
     }
+
+    void RegistryTestHelper::SetUp(AZStd::string_view path, bool value)
```

Comment:
suggestion: It might be better to pull out the `Set` call from the `SetUp` function as that'll make it easier to add other Set calls for different types (e.g. `float` or `Vector3` values)

Reply:
We can always do that later if it becomes useful. I just wanted to make the least complicated thing which would do what I needed.

Reply:
Okay fair enough, it just seems like we're bundling two fairly distinct things together, but it's trivial to split and refactor later if required you're right